build.sbt

```diff
@@ -833,6 +834,7 @@ lazy val `persistence-core` = (project in file("persistence/core"))
 
 lazy val `persistence-testkit` = (project in file("persistence/testkit"))
   .settings(runtimeLibCommon: _*)
+  .settings(noMima)
```

Comment:
This is a published artifact.

Reply:
Do we guarantee bincompat for testkits?

Reply:
I think we should. In the end, users code depend on it (and they can even create and publish extensions for our testkit, for example).